docs/content/preview/explore/change-data-capture/using-logical-replication/monitor.md

```diff
@@ -80,6 +80,12 @@ Provide information about CDC service in YugabyteDB.
 | cdcsdk_sent_lag_micros | `long` | The LAG metric is calculated by subtracting the timestamp of the latest record in the WAL of a tablet from the last record sent to the CDC connector. |
 | cdcsdk_expiry_time_ms | `long` | The time left to read records from WAL is tracked by the Stream Expiry Time (ms). |
 
+{{< note title="Note">}}
+
+The cdc service metrics are calculated for every tablet that is of interest for a replication slot. In the scenario where user is not interested in polling all the tables (and consequently all the tablets) in database, the metrics will be calculated considering the unpolled tablets until [cdcsdk_tablet_not_of_interest_timeout_secs](../../../reference/configuration/yb-tserver/#cdcsdk-tablet-not-of-interest-timeout-secs) interval.
```

Comment:
nit: Perhaps as a good practice, it is better to mention CDC in uppercase as we have mentioned in other places.